### PR TITLE
Removed __future__ imports

### PR DESCRIPTION
**Pull request by atombrella**
If Python 2 is not supported anymore, these shouldn't be here.

I left the 1.8 release notes untouched.

### CODE REVIEW

**Comment by BertrandBordage** [comment]
Thanks a lot @atombrella! :D
I was about to do it.
I scanned your pull request quickly, it seems fine, we can merge it once the test suite passes.

**Comment by gasman** [comment]
@BertrandBordage Please don't merge right now - I'm in the middle of dealing with #3918 :-)

**Comment by BertrandBordage** [comment]
Acknowledged, @gasman!

File: wagtail/wagtailadmin/datetimepicker.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
-from __future__ import absolute_import, unicode_literals
-
-
 # Adapted from https://djangosnippets.org/snippets/10563/
 # original author bernd-wechner
+
+
```
Comment (BertrandBordage) [inline wagtail/wagtailadmin/datetimepicker.py]: Why are these newlines added?

Comment (atombrella) [reply wagtail/wagtailadmin/datetimepicker.py]: `flake8` wasn't happy.

Comment (BertrandBordage) [reply wagtail/wagtailadmin/datetimepicker.py]: OK then.
I didn’t remember it forced spaces between comments & functions, that’s not specified in PEP8 as far as I remember.

Comment (atombrella) [reply wagtail/wagtailadmin/datetimepicker.py]: "Surround top-level function and class definitions with two blank lines." Doesn't it fall under this rule? I deleted the imports in the top, so the comments may be regarded as "top-level function".

**Comment by atombrella** [comment]
Hmm, Travis has some issues. But since it shouldn't be merged right away, I'll just let it sit.

I didn't change Github's default when I created the PR, and thus maintainers can push to my branch to trigger a new build on Travis when it's safe to merge.

File: wagtail/wagtailadmin/datetimepicker.py (hunk 1 above)
Comment (BertrandBordage) [reply wagtail/wagtailadmin/datetimepicker.py]: This doesn’t make sense to separate a comment from the function/class it’s about.
This rule is not an absolute rule to follow: for example, we don’t put two newlines at the end of the file after a class or function definition.
PEP8 itself shows some comments just above function definitions in this part: https://www.python.org/dev/peps/pep-0008/#code-lay-out

**Comment by gasman** [comment]
Thanks again @atombrella! Merged in f0eef2fc88043ba627b89125876f1cfe53b7fc51.

**Pull request closed without merging**